src/RegistrationManager.ts

```diff
@@ -390,16 +389,22 @@ export class RegistrationManager {
       transactions = transactions.concat(txHash);
     }
 
-    const portIncluded: boolean = this._config.app.url.indexOf(':') > 0;
+    const appConfig = config.get<AppConfig>('app');
+
+    const portIncluded: boolean = appConfig.url.indexOf(':') > 0;
```

Comment:
I think that the URL is always defined, isn't it? Otherwise, we should check for it

Reply:
I was thinking that we need to remove the port from the URL and merge it together, but I kept as it was before. 

But I dont understand what you are referring

Reply:
I meant, are we sure that the property `url` is defined in the `this._config.app` object?

Reply:
updated

Reply:
I think the URL is always defined due to the [default value](https://github.com/rsksmart/rif-relay-server/blob/ab39cefeee6d5fa8412d91fa7e5034b36aeafbf8/src/ServerConfigParams.ts#L78).